src/clj/rems/form_validation.clj

```diff
@@ -85,6 +85,17 @@
       {:field-id (:field/id field)
        :type     :t.form.validation/invalid-value})))
 
+(defn- multiselect-value-valid? [field]
+  (let [allowed-values (set (conj (map :key (:field/options field)) ""))]
+    (every? #(contains? allowed-values %)
+            (form/parse-multiselect-values (:field/value field)))))
+
+(defn- invalid-multiselect-error [field]
+  (when (= (:field/type field) :multiselect)
+    (when-not (multiselect-value-valid? field)
+      {:field-id (:field/id field)
+       :type     :t.form.validation/invalid-value})))
```

Comment:
Odd identation for us?

Reply:
Looks like it is not the only one.

Reply:
I've fixed them all whenever I bump into them. I agree it sometimes looks nice but is yet another exception in the formatting rules.